app/static/src/app/store/translations/sensitivityTab.ts

```diff
@@ -0,0 +1,16 @@
+interface SensitivityTabTranslations {
```

Comment:
is this going to be easier to mantain than the approach we take in hint/hintr etc? See the resource files here: https://github.com/mrc-ide/hintr/blob/master/inst/traduire/en-translation.json for example

Reply:
just to help locate the translation resources across the app